dayFour/ConstructorException.java

```diff
@@ -0,0 +1,29 @@
+package dayFour;
+
+class Base{
+	public Base() throws Exception {
+		System.out.println("Base constructor...");
+		throw new Exception("Exception in base class");
+	}
+}
+
+class Derived extends Base{
+	public Derived() throws Exception{
+		try{
+			Base b=new Base();
+		}catch(Exception e) {
+			System.out.println(e);
```

Comment:
Even though you have the catch statement in the derived class, the exception is not being handled. Can you update the code in such a way that, the exception that is being thrown from the base class gets handled in the derived class?

Reply:
I have added try catch code around the derived class object creation so that the exception can be handled

`try {
	Derived derived=new Derived();						
}catch(Exception exception) {
	System.out.println(exception.getMessage());
}`